config/default.yml

```
@@ -188,6 +188,11 @@ Performance/RedundantSortBlock:
   Enabled: true
   VersionAdded: '1.7'
 
+Performance/RedundantStringChars:
+  Description: 'Checks for places where `String#chars.<method_name>(args)` can be replaced by `String#<method_name>(args)`.'
```

The following explanation may be concise and sufficient than using placeholders.

```suggestion
  Description: 'Checks for redundant `String#chars`.'
```